arcade/text.py

```diff
@@ -252,32 +230,34 @@ def draw_text(text: str,
         text_sprite.width = image.width
         text_sprite.height = image.height
 
-        if anchor_x == "left":
-            text_sprite.center_x = start_x + text_sprite.width / 2
-        elif anchor_x == "center":
-            text_sprite.center_x = start_x
-        elif anchor_x == "right":
-            text_sprite.right = start_x
-        else:
-            raise ValueError(f"anchor_x should be 'left', 'center', or 'right'. Not '{anchor_x}'")
-
-        if anchor_y == "top":
-            text_sprite.center_y = start_y + text_sprite.height / 2
```

Comment:
This was the initial bug which wasn't allowing my `text_sprites` to be drawn with the correct `anchor_y`. Supposed to read `text_sprite.center_y = start_y - text_sprite.height / 2` which it does now in the fix.